### PR TITLE
version 1.0 released

### PR DESCRIPTION
**Pull request by narimiran**

### CODE REVIEW

File: jekyll/_posts/2019-09-23-version-100-released.md [hunk 1]
```diff
@@ -0,0 +1,102 @@
+---
+title: "Version 1.0 released"
+author: The Nim Team
+excerpt: "The Nim Team is very proud and happy to announce the much anticipated version 1.0 of the language."
```
Comment (ee7) [inline jekyll/_posts/2019-09-23-version-100-released.md]:
```suggestion
excerpt: "The Nim Team is very proud and happy to announce the much-anticipated version 1.0 of the language."
```

File: jekyll/_posts/2019-09-23-version-100-released.md [hunk 2]
```diff
@@ -0,0 +1,102 @@
...
+---
+
+
+Today is the day, the Nim Team is very proud
```
Comment (ee7) [inline jekyll/_posts/2019-09-23-version-100-released.md]:
```suggestion
Today is the day. The Nim Team is very proud
```

File: jekyll/_posts/2019-09-23-version-100-released.md [hunk 3]
```diff
@@ -0,0 +1,102 @@
...
+and happy to announce the much anticipated version 1.0 of the language.
```
Comment (ee7) [inline jekyll/_posts/2019-09-23-version-100-released.md]:
```suggestion
and happy to announce the much-anticipated version 1.0 of the language.
```

File: jekyll/_posts/2019-09-23-version-100-released.md [hunk 4]
```diff
@@ -0,0 +1,102 @@
...
+
+Nim was always focused on providing a compiled statically typed language
```
Comment (ee7) [inline jekyll/_posts/2019-09-23-version-100-released.md]:
```suggestion
Nim has always been focused on providing a compiled statically typed language
```

File: jekyll/_posts/2019-09-23-version-100-released.md [hunk 5]
```diff
@@ -0,0 +1,102 @@
...
+compile under any stable Nim 1.x version in the future.
+
+
+## What is included under the stability guarantee?
+
+Backwards compatibility covers only the stable fragment of the language,
+as defined by the [manual](https://nim-lang.org/docs/manual.html).
+
+The compiler still implements experimental features which are documented in the
+["experimental manual"](https://nim-lang.org/docs/manual_experimental.html),
```
Comment (ee7) [inline jekyll/_posts/2019-09-23-version-100-released.md]:
```suggestion
["experimental manual"](https://nim-lang.org/docs/manual_experimental.html).
```

File: jekyll/_posts/2019-09-23-version-100-released.md [hunk 6]
```diff
@@ -0,0 +1,102 @@
...
+these features are subject to changes which may be backwards incompatible,
```
Comment (ee7) [inline jekyll/_posts/2019-09-23-version-100-released.md]:
```suggestion
These features are subject to changes which may be backwards incompatible;
```

File: jekyll/_posts/2019-09-23-version-100-released.md [hunk 7]
```diff
@@ -0,0 +1,102 @@
...
+some of the features included under this umbrella are concepts,
+the `do` notation and a few others. There are also modules in the stdlib
+which are still considered unstable, these have been marked with an
```
Comment (ee7) [inline jekyll/_posts/2019-09-23-version-100-released.md]:
```suggestion
which are still considered unstable - these have been marked with an
```

File: jekyll/_posts/2019-09-23-version-100-released.md [hunk 8]
```diff
@@ -0,0 +1,102 @@
...
+these are not as fleshed out as we would like them to be.
+
+The standard library is also covered, as long as the module in question
+is clearly marked with a v1.0 tag in its documentation.
+
+
+## Exceptions to the rule
+
+We of course have to concede that there are exceptions.
+In certain serious cases, like for example when a security vulnerability is
```
Comment (ee7) [inline jekyll/_posts/2019-09-23-version-100-released.md]:
```suggestion
In certain serious cases, for example if a security vulnerability is
```

Be optimistic!

File: jekyll/_posts/2019-09-23-version-100-released.md [hunk 9]
```diff
@@ -0,0 +1,102 @@
...
+
+Over the years, more than 500 people contributed to the Nim codebase,
+implementing new features, fixing bugs and issues, writing documentation, and
+so on.
+The Nim team would like to thank all of you who helped us build Nim to become
+what it is today.
+
+We would also want to thank all people who have created Nimble packages,
+extending what is possible to do with Nim.
+The number of nimble packages has been steadily growing, and in August 2019 we
```
Comment (ee7) [inline jekyll/_posts/2019-09-23-version-100-released.md]:
```suggestion
The number of Nimble packages has been steadily growing, and in August 2019 we
```

**Comment by dom96** [comment]
Thanks for these suggestions @ee7, they're great!

**Comment by dom96** [comment]
Also, are we not including the changelog in this post?

**Pull request merged**